docs/API.md

```diff
@@ -1068,7 +1069,82 @@ for rErr := range minioClient.RemoveObjects(ctx, "my-bucketname", objectsCh) {
     fmt.Println("Error detected during deletion: ", rErr)
 }
 ```
+<a name="SelectObjectContent"></a>
+### SelectObjectContent(ctx context.Context, bucketName string, objectsName string, expression string, options SelectObjectOptions) *Events
+Parameters
+
+|Param   |Type   |Description   |
+|:---|:---| :---|
+|`ctx`  | _context.Context_  |Request context  |
+|`bucketName`  | _string_  |Name of the bucket  |
+|`objectName`  | _string_  |Name of the object |
+|`expression`  | _string_  |SQL expression|
+|`options` |  _SelectObjectOptions_  |  Query Options  |
+
+
+__Return Values__
+
+|Param   |Type   |Description   |
+|:---|:---| :---|
+|`Events` | _*Events_  | EventStream object that wraps the streaming for all Select events.  |
 
+```go
+	// Initialize minio client object.
+	minioClient, err := minio.New(endpoint, accessKeyID, secretAccessKey, useSSL)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	input := minio.SelectObjectInput{
+		RecordDelimiter: "\n",
+		FieldDelimiter:  ",",
+		FileHeaderInfo:  minio.CSVFileHeaderInfoUse,
+	}
+	output := minio.SelectObjectOutput{
+		RecordDelimiter: "\n",
+		FieldDelimiter:  ",",
+	}
+	opts := minio.SelectObjectOptions{
+		Type:   minio.SelectObjectTypeCSV,
+		Input:  input,
+		Output: output,
+	}
+	myReader , err := minioClient.SelectObjectContent(ctx, "sqlselectapi", "player.csv", "Select * from S3OBJECT WHERE last_name = 'James'", opts)
```

Comment:
This example doesnt look correct - SelectObjectContent returns a event stream according to the code, not a reader. Also the io.Pipe does not seem to be needed here.